server/.env

```diff
@@ -1,2 +1,5 @@
 TEAM_NAME="Shums, Michael"
 FLASK_ENV="development"
+APP_SECRET_KEY="random secret"
+GOOGLE_CLIENT_ID="734201335677-kakgc3np91nr45ss5j9f5l89v6b29h0n.apps.googleusercontent.com"
```

Comment:
hide this, git ignore this entire file

Reply:
**The following is what I did to add .env to .gitignore:**
1. `git rm --cached server/.env` in terminal
2. add `server/.env` to .gitignore